button-examples/Android/android-button-example.md

```diff
@@ -31,7 +31,14 @@ _**Note:** `<Button>` is auto-inflated as `<com.google.android.material.button.M
 
 ### Making buttons accessible
 
-Android's button component APIs support labeling for accessibility and are readable by most OCRs.
+Buttons support labeling for accessibility and are readable by most screen readers, such as TalkBack. By default a screen reader should identify the component as a button and read out the text label.
+
+Alternative text can be provided for the screen reader:
```

Comment:
I feel like we could also link to this website [https://support.google.com/accessibility/android/answer/7158690?hl=en](https://support.google.com/accessibility/android/answer/7158690?hl=en)
It says that content description is generally not needed for TextViews and its subclasses (like Buttons) but it also mentions an example where it's helpful (like when a play button changes to a pause button).

Reply:
Awesome, this is great. I'll update the PR.

Reply:
Done